docs/users/user_management_guide.md

```diff
@@ -0,0 +1,154 @@
+---
+
+# User management product guide
+
+User management is a fundamental aspect of any system.
+[[= product_name =]] offers a comprehensive and feature-rich user management system that allows organizations to efficiently manage their digital ecosystem.
+
+## What is user management
+
+User management refers to the process of granting, configuring, and controlling access for users by administrators.
+This encompasses the creation of user accounts, assigning roles and permissions, setting authentication methods, and managing user-related data.
+
+## Availability
+
+User management is available in all [[= product_name =]] versions.
+
+## How does user management work
+
+[[= product_name =]] simplifies user management with an intuitive and powerful system of accounts, Roles, Permissions, groups, and Segments.
+You can find all user groups and Users in the **Admin Panel** by selecting **Users**.
+Here, you can manage users, their relations, Roles, and Policies.
+
+![User's section](users_section.png)
+
+Here's how it works:
+
+- User accounts — create and manage user accounts. This includes capturing user information, such as name, email, and profile details.
+
+- Roles and Permissions — define Roles and assign Permissions to them. This ensures that users have appropriate access to content and functionalities. Roles can be customized to match the organization's specific needs.
+
+- Authentication methods — enable multiple authentication methods, including traditional username and password, OAuth, and external service logins. This flexibility allows organizations to adapt to various user authentication requirements.
+
+- User segmentation — segment users based on criteria such as demographics, behavior, or preferences. This segmentation enables personalized content delivery and targeted marketing.
+
+- Invitations — invite users to join a platform streamlining an onboarding process, sending invitations for exclusive content or events.
+
+- Customer groups — organize users into customer groups, which helps in delivering tailored experiences and content to specific segments.
+
+![User management](user_management.png)
+
+## Capabilities
+
+The detailed capabilities of [[= product_name_base =]] User Management,
+which provide organizations with the tools they need to deliver personalized, secure,
+and efficient user experiences while ensuring that user access and content delivery align with their business goals and strategies.
+
+### User Roles and Permissions
+
+[[= product_name_base =]] allows you to define custom user [Roles with granular Permissions](permission_overview.md), ensuring that users have access to only the specific parts of the system they need.
+Furthermore, you can create user groups to simplify Permission management.
+Assign multiple users to a group to ensure consistency and ease of access control.
+This helps maintain effortless security and control.
+
+**Permission:**
```

Comment:
Just an option to consider - list is more clear (in my opinion), and additionally, in the current form, when each sentence begins with "A" letter ('A Permission...', 'A Role...'), I see it as a letter numbered list.

Option:

- **Permission** - defines a specific action or access level that can be granted or denied within the system.

- **Role** - represents a collection of Permissions that can be assigned to users or user groups.
Roles streamline permission management by grouping related Permissions together.

- **Policy** - is a set of rules or conditions that determine under what circumstances a specific permission is granted or denied by applying Limitations.
Policies allow for fine-grained control of access based on various factors, such as user attributes or system states.